docs/developer-docs/emulator-setup-for-developers.md

```diff
@@ -75,22 +75,30 @@ When an update for RAIntegration is available, it will ask you if you want to up
 
 ## PPSSPP
 
-PPSSPP currently requires a development build for RAIntegration support:
+PPSSPP supports RAIntegration in its main builds now as of version 1.18. You can grab the latest version here:
 
-- **https://www.ppsspp.org/devbuilds/**
-
-Ensure that the file is named `RA_Integration-x64.dll` and place it in PPSSPP's main folder.
-
-![RAIntegration in PPSSPP folder](/public/ppsspp-raintegration-folder.png)
+- **https://www.ppsspp.org/download/**
 
-Then launch `PPSSPPWindows64.exe` and go to Settings -> Tools -> RetroAchievements. Here you can log in under the Account tab. Under the Developer Tools tab, you can enable RAIntegration. Enabling unofficial achievements may also be necessary in order to view them with the developer tools.
+After running the installer, launch PPSSPPP and go to Settings -> Tools -> RetroAchievements. Here you can log in under the Account tab. Under the Developer Tools tab, you can enable RAIntegration. Enabling unofficial achievements may also be necessary in order to view them with the developer tools.
 
 ![RAIntegration setting in PPSSPP](/public/ppsspp-raintegration-setting.png)
 
 Afterwards, exit PPSSPP and reopen it to load RAIntegration. RetroAchievements will have its own menu with the developer tools.
 
 ![RAIntegration menu in PPSSPP](/public/ppsspp-raintegration-menu.png)
 
+### Portable Install
+
+The installer from here downloads PPSSPP as an installer. If you want to avoid installing PPSSPP and run it as a portable application in its own folder, you can grab one of the development builds from here:
+
+- **https://www.ppsspp.org/devbuilds/**
+
+Ensure that the file is named `RA_Integration.dll` and place it in PPSSPP's main folder.
+
+![RAIntegration in PPSSPP folder](/public/ppsspp-raintegration-folder.png)
+
+After that, follow the instructions from above to Enable RAIntegration (for achievement development) and then reboot PPSSPP. You should get a success confirmation.
```

Comment:
```suggestion
After that, follow the instructions from above to Enable RAIntegration (for achievement development) and then restart PPSSPP. You should get a success confirmation.
```

Reply:
The overall wording changed a bit after pulling this into the section above, but I did make sure to use the term "restart" over "reboot"